app/lib/frontend/handlers/publisher.dart

```diff
@@ -31,8 +33,17 @@ Future<shelf.Response> getPublisherApiHandler(
 /// Handles requests for PUT /api/publisher/<publisherId>
 Future<shelf.Response> updatePublisherApiHandler(
     shelf.Request request, String publisherId) async {
-  // TODO: implement
-  return notFoundHandler(request);
+  final body = await request.readAsString();
+  final data = json.decode(body) as Map<String, dynamic>;
```

Comment:
If I send garbage instead of I'll get a 500, If I send `null` as body, I'll get a 500.

Users should never be able to provoke a 5xx error from us. As any 5xx is a bug in our code.

Reply:
++
should I wait on your code generator on that, or should I introduce something here first?